MAUI/DataForm/grouping.md

```diff
@@ -0,0 +1,386 @@
+            }
+        }
+    }
+
+{% endhighlight %}
+{% endtabs %}
+
+## Restricting the group expanding and collapsing
+
+The group being expanded or collapsed can be restricted by setting the [AllowExpandCollapse] to `false` in the [DataFormGroupItem].
+
+{% tabs %}
+{% highlight C# %}
+
+this.dataForm.GenerateDataFormItem += OnGenerateDataFormItem;
+
+    private void OnGenerateDataFormItem(object sender, GenerateDataFormItemEventArgs e)
+    {
+        if (e.DataFormGroupItem != null)
+        {
+            if (e.DataFormGroupItem.Name == "Name")
+            {
+                e.DataFormGroupItem.AllowExpandCollapse = false;
+            }
+        }
+    }
+
+{% endhighlight %}
+{% endtabs %}
+
+## Changing the group editors visibility
+
+The data form group visibility can be changed by using the [IsVisible] property of [DataFormGroupItem].
+
+{% tabs %}
+{% highlight C# %}
+
+this.dataForm.GenerateDataFormItem += OnGenerateDataFormItem;
+
+    private void OnGenerateDataFormItem(object sender, GenerateDataFormItemEventArgs e)
+    {
+        if (e.DataFormGroupItem != null)
+        {
+            if (e.DataFormGroupItem.Name == "Name")
+            {
+                e.DataFormGroupItem.IsVisible = false;
+            }
+        }
+    }
+
+{% endhighlight %}
+{% endtabs %}
+
+Here, the `Name` group will be hidden.
+
+## Changing the group header background
+
+The data form group header background can be changed by using the [HeaderBackground] property of [DataFormGroupItem].
+
+{% tabs %}
+{% highlight C# %}
+
+this.dataForm.GenerateDataFormItem += OnGenerateDataFormItem;
+
+    private void OnGenerateDataFormItem(object sender, GenerateDataFormItemEventArgs e)
+    {
+        if (e.DataFormGroupItem != null)
+        {
+            if (e.DataFormGroupItem.GroupName == "Name")
+            {
+                e.DataFormGroupItem.HeaderBackground = Brush.Yellow;
+            }
+        }
+    }
+
+{% endhighlight %}
+{% endtabs %}
+
+## Changing the group header text style
+
+The data form group header text style can be changed by using the [HeaderTextStyle] property of [DataFormGroupItem].
+
+{% tabs %}
+{% highlight C# %}
+
+this.dataForm.GenerateDataFormItem += OnGenerateDataFormItem;
+
+    private void OnGenerateDataFormItem(object sender, GenerateDataFormItemEventArgs e)
+    {
+        if (e.DataFormGroupItem != null)
+        {
+            if (e.DataFormGroupItem.GroupName == "Name")
+            {
+                e.DataFormGroupItem.HeaderTextStyle = new DataFormTextStyle
+                {
+                    TextColor = Colors.Violet,
+                    FontSize = 12,
+                    FontAttributes = FontAttributes.Italic,
+                };
+            }
+        }
+    }
+
+{% endhighlight %}
+{% endtabs %}
+
+## Changing the padding for group headers and editors
+
+The distance between editors and the data form’s borders or group headers can be changed by using [ItemsPadding] property of [DataFormGroupItem].
```

Comment:
The distance between editors and the data form’s borders or group headers can be changed by using the [ItemsPadding] property of the [DataFormGroupItem].

Reply:
Content changed